docs/recipes/precompiling-with-webpack.md

```diff
@@ -7,20 +7,26 @@ The AVA [readme](https://github.com/avajs/ava#transpiling-imported-modules) ment
 ###### webpack.config.js
 
 ```js
+const path = require('path');
+// To pre-compile folder with multiple test files, we can use glob
+// see http://stackoverflow.com/questions/32874025/how-to-add-wildcard-mapping-in-entry-of-webpack/34545812#34545812
+// const glob = require('glob');
+
 const nodeExternals = require('webpack-node-externals');
 
 module.exports = {
 	entry: ['src/tests.js'],
+	// entry: glob.sync('./test/**/*.js'),
```

Comment:
>  I'm not sure why the entry here is `src/tests.js`

Perhaps the entry should be changed to `tests/index.js`? 

> How are you using this?

I am using this in [elastic-builder](https://github.com/sudo-suhas/elastic-builder/blob/master/webpack.config.test.js). You can see the improved test duration here - https://travis-ci.org/sudo-suhas/elastic-builder/builds.

> Perhaps the globbing approach can be mentioned in the paragraph that follows the example code, with a link to the Stack Overflow post?

I wasn't entirely sure about how to present it. I am okay with this.